spec.html

```diff
@@ -1507,9 +1509,7 @@ <h1>Number::bitwiseNOT ( _x_ )</h1>
 
         <emu-clause id="sec-numeric-types-number-exponentiate" oldids="sec-applying-the-exp-operator">
           <h1>Number::exponentiate ( _base_, _exponent_ )</h1>
-          <p>
-            Returns an implementation-dependent approximation of the result of raising _base_ to the power _exponent_.
-          </p>
+          <p>The abstract operation Number::exponentiate takes arguments _base_ (a Number) and _exponent_ (a Number). It returns an implementation-dependent approximation to the result of raising _base_ to the power _exponent_, subject to the following requirements:</p>
```

Comment:
```suggestion
          <p>The abstract operation Number::exponentiate takes arguments _base_ (a Number) and _exponent_ (a Number). It returns an implementation-dependent approximation of the result of raising _base_ to the power _exponent_, subject to the following requirements:</p>
```

Reply:
The spec has only 2 occurrences of "approximation of" vs 40 of "approximation to", so I thought changing this "of" to "to" was justified. Is there something about this particular case that makes "of" the better choice?

Reply:
I'd personally prefer "approximation of" over "approximation to". But for this particular change, the wording was already "approximation of", so let's leave it like that and address the problem holistically later.

Reply:
Okay.